GenderPayGap.WebUI/Controllers/ApiForGpgAnalysisAppController.cs

```diff
@@ -81,8 +81,8 @@ public IActionResult ScopesAndReturns(string password)
                         // Organisations are allowed multiple SIC codes
                         // But, for this API, we just want 1 SIC code per organisation
                         // We use this method to keep this API consistent with the database query
-                        // TODO: we should really filter out Retired SIC codes

```

Comment:
Given this is a functionality change, will this have a knock-on effect anywhere else?

Reply:
@dulciejackson @Binney 
To be honest, this whole API wants to be thought about properly - I built it to match the data that was being returned by some old SQL stored procedures, but I think there's some data errors here - let's have a call to discuss